includes/admin/class-wcmp-export.php

```diff
@@ -1465,9 +1480,14 @@ private function saveOrderCollection(array $orderIds): array
             $this->orderCollection->push($order);
         }
 
-        $savedOrderCollection = $this->orderCollection->save();
+        try {
+            $savedOrderCollection = $this->orderCollection->save();
```

Comment:
Oei, dit vind ik geen fijne structuur, `save` impliceert iets wat gewoon `void` of `self` teruggeeft en dan evt het originele item (`orderCollection`) wijzigt... Maar dit heeft verder niks met je pr te maken hoor, ik wilde het alleen even zeggen voor de "toekomst" 😛